test-files/golden-tests/para-3.adoc

```diff
@@ -0,0 +1,51 @@
+= Reference
+:mrdocs:
+
+[#index]
+== Global namespace
+
+=== Functions
+[cols=2]
+|===
+| Name | Description 
+
+| <<#my_function,`my&lowbar;function`>> 
+| A function
+
+|===
+
+[#my_function]
+== my&lowbar;function
+
+A function
+
+=== Synopsis
+
+Declared in `&lt;para&hyphen;3&period;cpp&gt;`
+
+[source,cpp,subs="verbatim,replacements,macros,-callouts"]
+----
+void
+my&lowbar;function();
+----
+
+=== Description
+
+
+=== A heading&period;
```

Comment:
This seems wrong - the "A heading." should probably be one level "lower" than "Description" (but that's not really part of this PR, I suppose).

Reply:
Yes. This seems reasonable enough. There's a lot to think about regarding how this overview is structured.